frontend/web/components/modals/CreateFlag.js

```diff
@@ -86,10 +93,14 @@ const CreateFlag = class extends Component {
       isEdit: !!this.props.projectFlag,
       is_archived,
       is_server_key_only,
+      metadata: [],
+      metadataList: [],
```

Comment:
Im guessing metadata is the flag's metadata and list is organisation available? I think this would be automatically cleared up be reusing the metadata component.

Reply:
I will implement it

Reply:
Implemented